pages/common/ts-node.md

```diff
@@ -0,0 +1,28 @@
+# ts-node
+
+> Run TypeScript code directly, without any compiling.
+> More information: <https://www.npmjs.com/package/ts-node>.
+
+- Execute a TypeScript file without compiling (`node` + `tsc`).
+
+`ts-node {{path/to/file.ts}}`
+
+- Execute a TypeScript file without loading `tsconfig.json`:
+
+`ts-node --skip-project {{path/to/file.ts}}`
+
+- Display TS-Node help:
+
+`ts-node --help`
```

Comment:
Is this a helpful example? I have seen some comments that a help command, if included, is generally present as a last example.

Reply:
I have thus put it as the last example